### ISSUE TITLE
Not able to authenticate using file in VerneMQ. vmq-passwd: command not found

### ISSUE THREAD

**Issue by Mohitmedic** [vernemq/vernemq#1034]
I have compiled the code and trying to a demo user for authentication. But unable to add user because vmq-passwd command not found. How can I add user in VerneMQ authentication file.

### To help us save time and help you faster

Before opening a new issue, please make sure that isn't already an open (or
closed/resolved) issue concerning the problem. If unsure if older issues are
about the exact same thing, just open a new issue rather than posting in the old
one - but please reference the possibly related issues in the new issue.

Please do not insert images of text, but insert the text instead.

### Environment

- VerneMQ Version:
- OS:
- Erlang/OTP version (if building from source):
- VerneMQ configuration (vernemq.conf) or the changes from the default
- Cluster size/standalone:

### Expected behavior

- Please describe the expected behaviour.

### Actual behaviour

- Please include minimal steps to reproduce the issue, if possible.

**Comment by Mohitmedic** [comment]
Yes, Vernemq has been started. But when I trying to connect with client. I am getting error. Error Domain=NSPOSIXErrorDomain Code=61 "Connection refused"

Not able to find the reason where is the problem.

**Comment by larshesel** [comment]
This means that the MQTT listener is not reachable. Check your network and if the listener was really started on the server you're trying to reach.

**Comment by Mohitmedic** [comment]
I have restarted the verneMQ broker and check with ping command. its running fine. I installed Mosquitto broker on another server. There is no error with Mosquitto broker. But when I try to connect with verneMQ. I am getting error every time. I have check my network as well. I have enabled port as well. You can check attachment for the [port.
<img width="1125" alt="screenshot 2019-01-16 at 2 33 12 pm" src="https://user-images.githubusercontent.com/46735864/51237997-cc510d80-199b-11e9-83b5-f956dbc8d1b8.png">

**Comment by larshesel** [comment]
Hi, it's hard to know why it doesn't work - you have to go back and check the basics: check that VerneMQ is running on the server. Can you connect to the listener on the port when connecting from localhost on the server where VerneMQ is installed...
